HACKING.md

```diff
@@ -419,27 +473,36 @@ To run the spread tests via QEMU you need to create VM images in the
     $ mkdir -p ~/.spread/qemu
     $ cd ~/.spread/qemu
 
-Assuming you are building on Ubuntu 18.04 LTS (Bionic Beaver) (or a later 
-development release like Ubuntu 19.04 (Disco Dingo)), run the following to 
-build a 64-bit Ubuntu 16.04 LTS (Xenial Xerus) VM to run the spread tests on:
+Assuming you are building on Ubuntu 18.04 LTS ([Bionic Beaver](https://releases.ubuntu.com/18.04/)) 
+(or a later Ubuntu), run the following to build a 64-bit Ubuntu 16.04 LTS 
+(or later) with the correct `<release-name>` and `<release-version>`:
+
+    $ autopkgtest-buildvm-ubuntu-cloud -r <release-name>
+    $ mv autopkgtest-<release-name>-amd64.img ubuntu-<release-version>-64.img
+
+Use the following table as an example reference:
+
+| Release   | `<release-name>`  | `<release-version>`   |
+| -------   | --------------    |-------------------    |
+| [Xenial Xerus](https://releases.ubuntu.com/16.04/)  | xenial  | 16.04 |
+| [Bionic Beaver](https://releases.ubuntu.com/18.04/) | beaver  | 18.04 |
+| [Focal Fossa](https://releases.ubuntu.com/20.04/)   | focal   | 20.04 |
```

Comment:
this to me again seems like an example of documenting something we don't need to maintain here, I would rather us link to an upstream page which has this table - for example this does not mention impish or jammy here which we now have in our spread test environment and folks might want to use, so having to update this table all the time seems worse than just not having it here

Reply:
@anonymouse64, even better. I love it. Could you please refer me to the right location?

Reply:
an extremely cursory google result reveals https://wiki.ubuntu.com/Releases as a perfectly reasonable table of such values